infra/app/app-config/main.tf

```diff
@@ -20,7 +20,7 @@ locals {
   # If enabled, the networks associated with this application's environments
   # will have NAT gateways, which allows the service in the private subnet to
   # make calls to the internet.
-  has_external_non_aws_service = false
+  has_external_non_aws_service = true
```

Comment:
This might cost us $30/month per NAT gateway. I will log into AWS to see what they're projecting this thing to cost.